test/functional/terminal/buffer_spec.lua

```diff
@@ -242,7 +242,7 @@ describe('No heap-buffer-overflow when using', function()
     feed('$')
     -- Let termopen() modify the buffer
     feed_command('call termopen("echo")')
-    wait()
+    eq(2, eval('1+1')) -- check nvim still running
     feed_command('bdelete!')
```

Comment:
Should the "is alive" check go after this?

Reply:
The crash is expected at `termopen("echo")`. Though I would probably use sync `command()` if it doesn't matter